telehealth-example/Telehealth/PubNub/Theming/MessageInput/MessageInputTheme.swift

```diff
@@ -47,7 +47,7 @@ let messageInputComponentTheme = MessageInputComponentTheme(
     backgroundColor: UIColor(named: "MessageInput.TextView.BackgroundColor")!,
     textColor: UIColor(named: "MessageInput.TextView.TextColor")!,
     textFont: UIFont(name: "Poppins-Regular", size: 15)!,
-    usesStandardTextScaling: true,
+    usesStandardTextScaling: false,
```

Comment:
nitpick: if it not defaults to `false` can we just omit this parameter or is it still required?

Reply:
thats true, we can omit it